HFM Q2 milestone initial input files, readme, and video

mbkuhn opened the pull request.

Commit:
HFM Q2 milestone initial input files, readme, and video

Commit:
spelling

mpolimeno commented on exawind/OC6_platform_wave_tank/fixed_platform_irregular_waves/README.md:
```diff
@@ -0,0 +1,11 @@
+This benchmark problem uses the two-phase ExaWind solver, where water and air are simulated using the volume-of-fluid approach in both Nalu-Wind and AMR-Wind. The setup conforms to load case 3.3 from Phase Ia of the OC6 project (the definition document can be accessed here: https://www.nrel.gov/docs/fy22osti/80803.pdf). This load case involves a fixed platform within an irregular wave field, wher ...
```
There appears to be two typos here: "teh Maritime Research Institued" ("the" and 'Institute").

Also, perhaps a quick reference to Froude scaling, such as this one could be useful https://home.hvl.no/ansatte/gste/ftp/MarinLab_files/Litteratur/NTNU_Scaling_Laws.pdf ?

mpolimeno commented on exawind/OC6_platform_wave_tank/fixed_platform_irregular_waves/README.md:
```diff
@@ -1,10 +1,10 @@
-This benchmark problem uses the two-phase ExaWind solver, where water and air are simulated using the volume-of-fluid approach in both Nalu-Wind and AMR-Wind. The setup conforms to load case 3.3 from Phase Ia of the OC6 project (the definition document can be accessed here: https://www.nrel.gov/docs/fy22osti/80803.pdf). This load case involves a fixed platform within an irregular wave field, wher ...
+This benchmark problem uses the two-phase ExaWind solver, where water and air are simulated using the volume-of-fluid approach in both Nalu-Wind and AMR-Wind. The setup conforms to load case 3.3 from Phase Ia of the OC6 project (the definition document can be accessed here: https://www.nrel.gov/docs/fy22osti/80803.pdf). This load case involves a fixed platform within an irregular wave field, wher ...
```
"Institued" should be "Institute", I would think

Commit:
another spelling error

lawrenceccheung commented:
To keep things consistent with the other cases, I would create an `input_files` directory and put the `.inp` and `.yaml` in there.

mbkuhn commented:
I forgot my own system, whoops

Commit:
move input files to directory

lawrenceccheung commented on exawind/OC6_platform_wave_tank/fixed_platform_irregular_waves/README.md:
```diff
@@ -0,0 +1,11 @@
+
+The AMR-Wind computational domain extends from -10m to 10m in the x direction, which is only a portion of the length of the original wave tank, while it extends from -2m to 2m in y, the full width of the tank, and from -3.6m to 2.4m in z, reaching the full depth of the tank and providing enough room around the vertical extent of the platform. Two levels of refinement are added, and these are targ ...
+
+The Nalu-Wind computational mesh is tetrahedral and adheres to the mesh resolution recommendations in section 3.2.3 of this paper: https://www.sciencedirect.com/science/article/pii/S0029801822013580. (Need way to access mesh)
```
The mesh files are now available at https://github.com/Exawind/exawind-benchmarks/releases/download/mesh_assets/oc5-resmatch.e.tar.gz and https://github.com/Exawind/exawind-benchmarks/releases/download/mesh_assets/modes_HOS_SWENSE.dat.bz2.  I might have to edit the `downloadmeshes.py` to account for them, but we can worry about that later.

lawrenceccheung commented on exawind/OC6_platform_wave_tank/fixed_platform_irregular_waves/README.md:
```diff
@@ -0,0 +1,11 @@
+
+This case is currently being run to reach a minimum duration of 9 minutes, but here is a low-frame-rate video from the first minute, showing the evolution of the air-water interface within the Nalu-Wind portion of the domain.
+
+![OC6 fixed platform early](/Users/mkuhn/exawind-benchmarks/exawind/OC6_platform_wave_tank/floater_interface_nalu.mp4)
```
This location of the video looks like it's relative to a location on your local machine.  I think you can do a link to an mp4 video in markdown in github and have it show up, but not sure if it'll appear on a sphinx webpage.

mbkuhn replied:
good catch. I'm going to try with a relative link, then hopefully when we port it to sphinx we can copy the video file to the same relative location.

lawrenceccheung commented on exawind/OC6_platform_wave_tank/fixed_platform_irregular_waves/README.md:
```diff
@@ -0,0 +1,11 @@
+This benchmark problem uses the two-phase ExaWind solver, where water and air are simulated using the volume-of-fluid approach in both Nalu-Wind and AMR-Wind. The setup conforms to load case 3.3 from Phase Ia of the OC6 project (the definition document can be accessed here: https://www.nrel.gov/docs/fy22osti/80803.pdf). This load case involves a fixed platform within an irregular wave field, wher ...
```
I would add a markdown title to the top of this document, like:  
```markdown
# OC6 Floating platform simulation
```
I think Sphinx will use it to generate the right title headers in the html

lawrenceccheung commented on exawind/OC6_platform_wave_tank/fixed_platform_irregular_waves/README.md:
```diff
@@ -0,0 +1,11 @@
+
+The irregular wave field forced into the AMR-Wind domain is first generated by the high-order spectral solver HOS-NWT, which produces a file containing a time series of wave modes. These are read into AMR-Wind, transformed, and interpolated onto the computational mesh during the ExaWind simulation. When selecting the wave setup for HOS-NWT, a variety of approaches were evaluated, including iterat ...
+
+This case is currently being run to reach a minimum duration of 9 minutes, but here is a low-frame-rate video from the first minute, showing the evolution of the air-water interface within the Nalu-Wind portion of the domain.
```
Maybe we can also add that more complete run instructions will be added later, with executable details.  That might warn off anybody who's might attempt to run this case now, without the right code changes in ExaWind.

lawrenceccheung reviewed (changes requested):
Thanks @mbkuhn!  Just a couple of minor things to the README.  I'll also put this into IR so that it can be approved for publishing on the website too.

Commit:
requested changes
- mesh and modes file links
- video path update
- markdown title
- caveat for future details
